packages/builder/test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "include": [
+    "../src",
+    "../../@types",
+    "../test"
```

Comment:
Eslint does only check that sources are included into the project.
So this config works as long as it isn't used to compile the files.

Reply:
so, maybe it will be easier to mute that check in `eslintrc`? Because now you are adding all files, so obviously all are included in the project, so the rule doesn't make sense any more.

Reply:
This is not what I wanted to address.
Eslint complains if `test` is not added to `tsconfig.json`. 
And compiler overrides the sources if it finds the same ones in `test/`.
So if we have this separate `tsconfig.json` file and it's only used by eslint then everything works:
- Compiler compiles as before
- Eslint can check both: `src` and `tests`.

Reply:
OK, but probably we will rollback it in the future once we will support `tests` compilation.

Reply:
BTW, instead of writing `../test` we should write `./`